$NetBSD: patch-configure,v 1.3 2025/03/12 22:46:21 gdt Exp $

Don't set _POSIX_C_SOURCE, because visibility defines hide other
things, and it breaks NetBSD's <netinet/ip.h>, and also on macOS.

Reported upstream by email 20240710, 20250312.

--- configure.orig	2025-03-12 19:27:58.000000000 +0000
+++ configure
@@ -20636,7 +20636,7 @@ fi
 
 # C99 + POSIX extras
 
-  for stp_ac_arg in -D_POSIX_C_SOURCE=200809L -std=c99 ; do
+  for stp_ac_arg in -std=c99 ; do
 
   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking if ${CC} supports ${stp_ac_arg}" >&5
 printf %s "checking if ${CC} supports ${stp_ac_arg}... " >&6; }
